.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report
+about: 'Create a report to help us solve an unexpected issue. '
+title: 'Bug: [name of bug]'
+labels: ''
+assignees: ''
+---
+
+This project was created not to be 'production ready' at Enterprise standard, but instead be a fun and interactive project that can also be used to learn together on stream. 
+
+Genuine bugs may happen and these are appreciated, but your opinion of how the code could be written differently or 'better' is not a bug.
```

Comment:
> opinion of how the code could be written differently or 'better'
Indeed not a bug, because this describes a refactor request. IMHO this can be rephrased to a measurable/objective metric to avoid discussions in issues about whether it constitutes opinion / readability improvement, as opposed to a simple metric of "Is there a failing test case?"